tests/cli/test_cli.py

```diff
@@ -164,7 +164,7 @@ def get_info(uuid, key):
     return _run_command([cl, 'info', '-f', key, uuid])
 
 
-def wait_until_state(uuid, expected_state, timeout_seconds=1000):
+def wait_until_state(uuid, expected_state, timeout_seconds=1000, exclude_final_states=set()):
```

Comment:
let's simplify to a boolean

```suggestion
def wait_until_state(uuid, expected_state, timeout_seconds=1000, exclude_final_states=False):
```